WikiWatershed/model-my-watershed
# Feature/layerpicker

## Overview

Aggregate of all the work we've done thus far to begin implementing the new wireframes:
- Adds new layerpicker
- Adds breadcrumb navigation
- Removes back buttons

## Testing Instructions
Run through the app and make sure the rebase didn't break anything
- Add/remove layers
- navigate with breadcrumbs

Connects #1737
